pages/common/plesk.md

```diff
@@ -0,0 +1,28 @@
+# plesk
+
+> Plesk hosting control panel CLI interface.
+> More information: <https://docs.plesk.com>.
+
+- Generate an auto login link for the admin user:
+
+`plesk login`
+
+- Show product version information:
+
+`plesk version`
+
+- List all hosted domains:
+
+`plesk bin domain --list`
+
+- Start watching for changes in the `panel.log` file:
+
+`plesk log panel.log`
```

Comment:
Can this be any file?  Or is required that it be `panel.log`?

Reply:
No, it can't be any file. The utility helps to "tail" easily several main (for control panel) log files. They are located in different places and utility helps to discover them in an easy manner. The utility goes with bash-completion, so the user can discover other options just by hitting "Tab".

Reply:
How many options are there?  If there are a few, than we could list them all right here, like this: `{{panel.log|abc.log|def.log}}`.

Reply:
Not so many. Ok, let's do it.

Reply:
I realized that description becomes weird in such a case. Let me show. The options are:  --all, access_log, error_log, maillog, and panel.log. "--all" have different semantic - it means we start watching all the log files. The most popular option is "panel.log". It is used in 90% of cases. So the description "Start watching for changes in the log file:" is a little bit misleading. Moreover, the most popular option (panel.log) becomes not so clear if I change the example. I prefer to keep the original example.

Reply:
Interesting.  Sounds good.